src/components/CurrentWeather.vue

```diff
@@ -30,35 +42,79 @@
 <script lang="ts">
```

Comment:
Please notice in vue docs, they suggest to order parts of component as `script`, `template`, `style`